### PR TITLE
Home page style

### PR DESCRIPTION
**Pull request by LorenzoEvans**
# Description

Tweaking scss for smooth scaling of home page content (text, images, containers) at most sizes.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [ x ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ x ] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -417,4 +417,6 @@ class App extends Component {
 
 // please update your components and Routes as needed
 
-export default withAuthentication(App);
+// export default withAuthentication(App);
+
+export default App ;
```
Comment (TheGameFreak720) [inline client/src/App.js]: I think we need this to be withAuthentication but if it's not needed please remove the comment

File: client/src/components/Home/Featured/Featured.js [hunk 2]
```diff
@@ -11,12 +11,12 @@ function Featured(props) {
   }
   return (
     <div className="card" onClick={() => props.clickHandler(props.username)}>
-      <img src={props.image} alt="project" />
+      <img className="img-responsive" src={props.image} alt="project" />
       <div className="info-container">
-        <h2>{props.title}</h2>
+        <h2 className="proj-title" >{props.title}</h2>
```
Comment (TheGameFreak720) [inline client/src/components/Home/Featured/Featured.js]: like the naming convention

File: client/src/styles/card.scss [hunk 3]
```diff
@@ -1,44 +1,70 @@
...
+  }
+
+  .proj-title {
+   font-size: 2rem ;
+   width: 100% ;
+   overflow: hidden;
+   white-space: nowrap;
+   text-overflow: ellipsis ;
+   text-align: center ;
+  }
```
Comment (TheGameFreak720) [inline client/src/styles/card.scss]: That's some good shit. We can use this through all the application

**Comment by TheGameFreak720** [comment]
I would say as a change adding that spacing underneath the image. They are too close together but apart from that good job man

image

**Comment by LorenzoEvans** [comment]
> I would say as a change adding that spacing underneath the image. They are too close together but apart from that good job man
> 
> image

Like, more spacing under the Popular Makers image?

**Comment by TheGameFreak720** [comment]
spacing inside the card. You need to have spacing between the title, username, button, and the stars.

**Pull request merged**